vpr/src/base/atom_netlist_utils.cpp

```diff
@@ -38,7 +38,7 @@ bool is_removable_input(const AtomNetlist& netlist, const AtomBlockId blk, std::
 bool is_removable_output(const AtomNetlist& netlist, const AtomBlockId blk, std::string* reason = nullptr);
 
 //Attempts to remove the specified buffer LUT blk from the netlist. Returns true if successful.
-bool remove_buffer_lut(AtomNetlist& netlist, AtomBlockId blk, int verbosity);
+bool remove_buffer_lut(AtomNetlist& netlist, std::set<AtomPinId> clock_drivers, AtomBlockId blk, int verbosity);
```

Comment:
Since I suggest generalizing this to capture *all* merged nets we don't need to pass in the clock drivers specifically.

Reply:
Done